core/src/main/java/org/testcontainers/containers/traits/ClasspathBind.java

```diff
@@ -0,0 +1,39 @@
+package org.testcontainers.containers.traits;
+
+import org.testcontainers.containers.BindMode;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.TestContainer;
+import org.testcontainers.utility.SelfReference;
+
+import java.net.URL;
+import java.util.Optional;
+
+public class ClasspathBind<SELF extends TestContainer<SELF>> extends FileSystemBind<SELF> {
```

Comment:
I decided to create ClasspathBind to make it's possible to use `.with(new ClasspathBind("file.txt", "/file.txt", ...))` notation